openlane/flows/classic.py

```diff
@@ -284,18 +291,20 @@ class Classic(SequentialFlow):
         Odb.RemovePDNObstructions,
         OpenROAD.STAMidPNR,
         OpenROAD.RepairDesignPostGPL,
-        Odb.DiodesOnPorts,
-        Odb.HeuristicDiodeInsertion,
         OpenROAD.DetailedPlacement,
         OpenROAD.CTS,
         OpenROAD.STAMidPNR,
         OpenROAD.ResizerTimingPostCTS,
         OpenROAD.STAMidPNR,
         OpenROAD.GlobalRouting,
         OpenROAD.RepairDesignPostGRT,
+        Odb.DiodesOnPorts,
+        Odb.HeuristicDiodeInsertion,
```

Comment:
need a global routing step here in case repairantennas is off

Reply:
I've elected to, similar to the other steps, simply legalize within those steps themselves until we have composite steps.